server/protocols/jmap/src/main/java/org/apache/james/jmap/UserProvisioningFilter.java

```diff
@@ -75,11 +73,11 @@ void createAccountIfNeeded(MailboxSession session) {
         try {
             User user = session.getUser();
             if (needsAccountCreation(user)) {
-                createAccount(user);
+                usersRepository.addUser(getUsername(user), generatePassword());
```

Comment:
Ok, account is a jmap thing, I agree that it's not ok here.
Nonetheless, createUser(user) will always be easier to read than `usersRepository.addUser(getUsername(user), generatePassword());`